Runtime/Editor/LootLockerExtensionCalls.cs

```diff
@@ -0,0 +1,143 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.NetworkInformation;
+using LLlibs.ZeroDepJson;
+using LootLocker.Extension.Requests;
+using LootLocker.Requests;
+using UnityEditor.VersionControl;
+using UnityEngine;
+
+namespace LootLocker.Admin
+{
+    public partial class LootLockerAdminManager
+    {
+                
+        public static void AdminLogin(string email, string password, Action<LoginResponse> onComplete)
+        {
+            if (string.IsNullOrEmpty(email) || string.IsNullOrEmpty(password))
+            {
+                Debug.Log("Input is invalid!");
+                return;
+            }
+
+            AdminLoginRequest request = new AdminLoginRequest();
+            request.email = email;
+            request.password = password;
+
+            AdminLogin(request, onComplete);
+        }
+
+        public static void AdminLogin(AdminLoginRequest data, Action<LoginResponse> onComplete)
+        {
+            EndPointClass endPoint = LootLockerEndPoints.adminExtensionLogin;
+
+            string json = LootLockerJson.SerializeObject(data);
+
+            LootLockerServerRequest.CallAPI(endPoint.endPoint, endPoint.httpMethod, json,
```

Comment:
Maybe make a helper method for CallAPI in this class that just takes all the arguments and pass them on but set callerRole = Admin centrally instead?

Reply:
Done :)